webview/src/experiments/components/table/styles.module.scss

```diff
@@ -243,8 +243,8 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
       .bullet {
         &::after {
           content: '';
-          height: 100%;
-          top: 0;
+          height: 105%;
```

Comment:
Is it possible to use `calc` here? (same for line256) Like `calc(100% + 5px)`. I guess the 5% here is so the line can overflow the cell?  I feel like `calc` makes this more obvious and more precise.

Reply:
Good idea!